plugins/org.jboss.reddeer.jface/src/org/jboss/reddeer/jface/handler/ActionContributionItemHandler.java

```diff
@@ -1,96 +0,0 @@
-/******************************************************************************* 
- * Copyright (c) 2016 Red Hat, Inc. 
- * Distributed under license by Red Hat, Inc. All rights reserved. 
- * This program is made available under the terms of the 
- * Eclipse Public License v1.0 which accompanies this distribution, 
- * and is available at http://www.eclipse.org/legal/epl-v10.html 
- * 
- * Contributors: 
- * Red Hat, Inc. - initial API and implementation 
- ******************************************************************************/ 
-package org.jboss.reddeer.jface.handler;
-
-import org.eclipse.jface.action.ActionContributionItem;
-import org.eclipse.jface.action.IAction;
-import org.jboss.reddeer.common.logging.Logger;
-import org.jboss.reddeer.common.util.Display;
-import org.jboss.reddeer.common.util.ResultRunnable;
-import org.jboss.reddeer.core.exception.CoreLayerException;
-
-/**
- * Contains methods for handling UI operations on {@link ActionContributionItem}
- * widgets.
- * 
- * @author rawagner
- *
- */
-public class ActionContributionItemHandler {
```

Comment:
This was used to handle view menus but I rewrite the logic for view menus in previous commits and ActionContributionItem is not used anymore anywhere